packages/components/src/components/post-icon/post-icon.tsx

```diff
@@ -6,19 +6,26 @@ import { Component, Host, h, Prop, State } from '@stencil/core';
   shadow: true,
 })
 export class PostIcon {
-  @Prop() name: number;
+  @Prop() name: string;
   @Prop() base?: string;
+  @Prop() scale?: string;
+  @Prop() rotate?: string;
```

Comment:
Could be worth adding doc block comments here. They will be generated as description in the .md file and picked up by the stencil-storybook plugin.